subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1620,8 +1627,13 @@ static int light_ctrl_srv_start(struct bt_mesh_model *model)
 
 	atomic_set_bit(&srv->flags, FLAG_STARTED);
 
+	if (IS_ENABLED(CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG) && srv->reg) {
+		srv->reg->measured = 0;
```

Comment:
> If we change something with regulator startup, this will be easily missed.

This should be covered by a test.

Reply:
This is to make sure that the regulator uses random values before it receive it's first Ambient light value. It can receive other values than Amb light level, and until it receive its first value the spec want to make sure that the value is 0. Is not that the purpose of that sentence in the spec.

Reply:
The ambient light value is only used together with the regulator. The regulator is off until the lc server receives first ambient light level report: https://github.com/nrfconnect/sdk-nrf/blob/e28de89573274076beae9d0afc558288383be87b/subsys/bluetooth/mesh/light_ctrl_srv.c#L987-L996

Reply:
I also see that the regulator is started after the "measured" value is set. In one way I want to keep this as it is, to show that the regulator measured value is set to 0 when powered on and provisioned. I think this is more according to spec than. If you both agree that this should be removed, I have not problem removing this lines.

Reply:
We already have a test for this so I don't think this code is needed: https://github.com/nrfconnect/sdk-nrf/blob/e28de89573274076beae9d0afc558288383be87b/tests/subsys/bluetooth/mesh/light_ctrl/src/main.c#L1059-L1060

Reply:
Sorry, no, it is not this one, but anyway. We could write one more test to cover this, but I don't think this is needed. This code won't do anything anyway, so it is better to keep the code clean.

Reply:
What do you think @omkar3141? If you agree I can remove it.